frontend/app/module/election/page/ElectionStatusPage.tsx

```diff
@@ -41,9 +189,61 @@ export function ElectionStatusPage() {
           </Button>
         </Alert>
       )}
-      <main>
-        <ElectionStatusProgress />
-        <article>Placeholder</article>
+      <main className={cls.statusMain}>
+        <div className={cls.statusTitle}>
+          <h2 id="status-title">{t("election_status.main_title")}</h2>
+          {/* TODO: Add button onClick to Create Polling Station page */}
+          <Button size="md" variant="secondary" leftIcon={<IconPlus />}>
+            {t("election_status.add_polling_station")}
+          </Button>
+        </div>
+        <div className={cls.statusSection}>
+          <Progress>
+            <div id="shortcuts" className="column">
+              <h2 className="mb-0">{t("shortcuts")}</h2>
+              {statusCategories.map((cat) => {
+                return (
+                  <span
+                    className="item"
+                    key={`item-progress-${categoryColorClass[cat]}`}
+                    id={`item-progress-${categoryColorClass[cat]}`}
+                  >
+                    <Icon icon={<IconDot />} size="sm" color={categoryColorClass[cat]} />
+                    {t(`status.${cat}`)} ({categoryCounts[cat]})
+                  </span>
+                );
+              })}
+            </div>
+            <div id="progress" className="column">
+              <h2>{t("progress")}</h2>
+              <ProgressBar key="all" id="all" data={progressBarData} spacing="small" />
+            </div>
+          </Progress>
+          <article className={cls.statusArticle}>
+            {statusCategories
+              .filter((cat) => categoryCounts[cat] !== 0)
```

Comment:
personal preference, do the filtering outside the return, eg. `const categories = statusCategories.filter()`
it is easier to do more stuff to it later

Reply:
Changed in 8092d5d90d89d8bdec7e01b9e81c8a4ba492160a